src/Templator.ts

```
@@ -160,0 +1,159 @@
+                }
+            },
+            markdown_escape() {
+                return function(text: string, render: any) {
+                    return render(text).replace(/\(/g, "&lpar;").replace(/\)/g, "&rpar;");
+                }
+            },
+            query_filter() {
+                const view = this;
+                return function(text: string, render: any) {
+                    // query = {{`tag?sort(by)/filter/`}}
+                    const query_extract = /^{{`(?<tag>\w+)(?:\?(?<fn>\w+)\((?<args>[\w\s,]*)\))*\/(?<filter>[^]*)\/`}}/;
```

Biome (2.1.2)

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)